packages/user-profile-widget/src/lib/widget/mixins/initMixin/initComponentsMixins/initWidgetRootMixin.ts

```diff
@@ -0,0 +1,81 @@
+import {
+  compose,
+  createSingletonMixin,
+  createTemplate,
+} from '@descope/sdk-helpers';
+import {
+  descopeUiMixin,
+  initElementMixin,
+  initLifecycleMixin,
+  loggerMixin,
+} from '@descope/sdk-mixins';
+import { fetchWidgetPagesMixin } from '../../fetchWidgetPagesMixin';
+import { stateManagementMixin } from '../../stateManagementMixin';
+
+export const initWidgetRootMixin = createSingletonMixin(
+  <T extends CustomElementConstructor>(superclass: T) =>
+    class InitWidgetRootMixinClass extends compose(
+      loggerMixin,
+      initLifecycleMixin,
+      descopeUiMixin,
+      initElementMixin,
+      fetchWidgetPagesMixin,
+      stateManagementMixin,
+    )(superclass) {
+      async #initWidgetRoot() {
+        const template = createTemplate(
+          `
+          <descope-container data-editor-type="container" direction="column" id="ROOT" space-between="md" st-horizontal-padding="1rem" st-vertical-padding="1rem" st-align-items="safe center" st-justify-content="safe center" st-host-width="100%" st-gap="1rem">
```

Comment:
I guess we want to take this from the mock file like in all other widgets.

Reply:
Fixed